crypto/fipsmodule/rand/rand.c

```diff
@@ -64,14 +64,24 @@
 // reseeding.
 
 #if defined(BORINGSSL_FIPS)
+
+#define FIPS_ENTROPY_SOURCE_JITTER_CPU 1
+
+#if !defined(FIPS_ENTROPY_SOURCE_JITTER_CPU)
```

Comment:
Won't this always be true due to line 68?

Reply:
Yes, for now it will! But later this will be demoted to a choice. Just paranoid.